.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vscode-dvc-demo"]
+	path = vscode-dvc-demo
```

Comment:
You're right, it's not because the repo is called vscode-dvc-demo that the path needs to be called the same thing. That'll definitely simplify the diff. I'll make the change.